### PR TITLE
Feat: remove apiserver component from the chart

### LINKED ISSUE #2801: Refine Vela Core Chart
This issue is to collect feedback on current vela-core chart which is too complicated. Let's share the problems you saw on the chart.

### PR DESCRIPTION
**Pull request by barnettZQG**
### Description of your changes

The apiserver is no longer deployed by the chart.

Fixes #2801

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Comment by wonderflow** [comment]
This is a break change, you should add a migrate docs for the existing users.

**Comment by barnettZQG** [comment]
> This is a break change, you should add a migrate docs for the existing users.

 Before release, we provide the upgrade document in combination with other changes.

File: pkg/stdlib/pkgs/query.cue [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
-			components?: [...string]
-		}
-	}
-	list?: [...{
-		cluster:   string
-		component: string
-		revision:  string
-		object: {...}
-	}]
+  #do:       "listResourcesInApp"
```
Comment (FogDong) [inline pkg/stdlib/pkgs/query.cue:2]: Are these changes related to our chart?
